amarachukwu-agbo/hello-books
# #151748633 Email notification for users on borrow/return request status

#### What does this PR do?
Send email to users when their request to borrow or return a book is accepted/declined.

#### Description of Task to be completed?
- Setup node-mailer for sending emails
- Include function to send email to users
- Update .env.example file to reflect new environment variables

#### How should this be manually tested?
Clone the repo and set up the application as explained in the README. Start the server using ```npm run start:server```. Go to Postman on  ```POST 127.0.0.1:3000/api/v1/users/:userId/borrow/:bookId``` and borrow a book using a user account. Accept the request on Postman on the route ```POST 127.0.0.1:3000/api/v1/users/:userId/borrow/:bookId``` using an admin account and an email should be sent to the user account stating the status of the request.

#### Any background context you want to provide?
Making a request to borrow a book and accepting the request both require authentication so login to the application on ```POST 127.0.0.1:3000/api/v1/users/login``` and get the token assigned to use in the Authorization header when making the requests.

#### What are the relevant pivotal tracker stories?

#151748633

#### Screenshots (if appropriate)

N/A

#### Questions: N/A
